ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/ExperimentalPreviewRevenueCatUIPurchasesAPI.kt

```diff
@@ -0,0 +1,19 @@
+package com.revenuecat.purchases.ui.revenuecatui
+
+/**
+ * This annotation marks the experimental preview of the RevenueCat UI Purchases API.
+ * This API is in a preview state and has a very high chance of being changed in the future.
```

Comment:
Maybe something like this too:
```suggestion
 * This API is in a preview state and has a very high chance of being changed in the near future.
 
 Once RevenueCat UI is stable, this annotation will be removed.
```